Packs/mnemonicMDR/pack_metadata.json

```diff
@@ -36,4 +35,4 @@
         "MDR"
     ],
     "githubUser": []
```

Comment:
Please update this field.

Reply:
I thought this field was optional. Could you explain its function, please? I am a little uncertain about the meaning:

> (Optional) List of Github usernames to receive notification in the PR in case pack files were modified.

Reply:
Whenever a file in this pack would change, the users that are configured inside the `githubUser` field will be notified.
It is highly encouraged for partners to fill this field in order to keep track of their packs.